src/pages/my_page/user_profile/MyProjects.tsx

```diff
@@ -21,10 +23,17 @@ export const MyProjects = () => {
 
   const [page, setPage] = useState(1);
   const [rowsPerPage, setRowsPerPage] = useState(ROWS_PER_PAGE_OPTIONS[0]);
+  const [projectSort, setProjectSort] = useState(projectSortOptions[0]);
+
+  const projectsQueryConfig: ProjectsSearchParams = {
+    participant: userCristinId,
+    orderBy: projectSort.orderBy,
+    sort: projectSort.sortOrder,
+  };
 
   const projectsQuery = useQuery({
```

Comment:
Kan endre dette til å bruke egen hook, ved interesse, men da blir det noen endringer i `useFetchProjects` og komponenter som bruker den

Reply:
Vi har vel generelt tatt i bruk hooks for flere `fetch`-operations, så kanskje like greit å gjøre det her også?

Reply:
Kanskje best å ta det i en egen PR uansett i så fall, siden det vil påvirke filer som ikke har noe med dette å gjøre direkte